test/extensions/filters/http/common/fuzz/filter_fuzz_test.cc

```diff
@@ -22,9 +28,6 @@ DEFINE_PROTO_FUZZER(const test::extensions::filters::http::FilterFuzzTestCase& i
         static const std::vector<absl::string_view> registered_names = Registry::FactoryRegistry<
             Server::Configuration::NamedHttpFilterConfigFactory>::registeredNames();
         // Exclude filters that don't work with this test. See #20737
```

Comment:
nit: comment should also move up.

Reply:
Mhhh, really, actually is the code following doing the filtering of the tests that don't work. Therefore I left the comment there. Having it in front of the now dual-use exclusion list would be misleading, wouldn't it?